### PR TITLE
Repo Admin

### LINKED ISSUE #3: Organise the repo
We should organise the repo a little more, e.g.:
```
notebooks/
data/
  MDCK_H2B_GFP_movie.tif
  annotation_02-08-2021--10-33-59.zip
  ...
assets/
  cell_cycle_states.png
  ...
```

### PR DESCRIPTION
**Pull request by KristinaUlicna**
Closes #3

Updated README.md files in the main tree + /notebooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
+
+### How to navigate this repository
+
+We provide a detailed walk-through for annotating live-cell microscopy images and training CNN model to infer classification labels on previously unseen images. Here is an overview of the entire process:
+
+![Protocol Pipeline](/assets/protocol_pipeline.png)
+
+For more detailed instructions on how to annotate your microscopy data, train the CNN classifier and infer labels on previously unseen images, please refer to [this step-wise manual](/notebooks/README.md "Protocol Methods").
+
+Please use these links to proceed with the training and inference of your CNN models in the Google Colab environment:
```
Comment (quantumjot) [inline README.md]: This is great!

File: data/annotation_03-08-2021--13-39-57/annotation_03-08-2021--13-39-57.json [hunk 2]
```diff
@@ -0,0 +1,325 @@
+{
+  "shape": 64,
+  "GFP": {
+    "filename": "../data/MDCK_H2B_GFP_movie.tif"
+  },
+  "coords-0": [
```
Comment (quantumjot) [inline data/annotation_03-08-2021--13-39-57/annotation_03-08-2021--13-39-57.json]: Can you get rid of all of the uncompressed files?

File: notebooks/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# CNN-Annotator Repository Structure
+
+![Protocol Pipeline](../assets/repository_structure.png)
+
+### Annotating CNN training images using *Napari*:
```
Comment (quantumjot) [inline notebooks/README.md]: This is great too. I wonder if we can export the paper itself as markdown when we're finished with it?

**Review by quantumjot — CHANGES_REQUESTED** [review]
Looks great @KristinaUlicna .  Can we just remove the uncompressed folder/files in `data/annotation_03-08-2021--13-39-57/ `? Then I'll merge it

File: README.md [hunk 4]
```diff
@@ -1,15 +1,33 @@
-# CNN Annotator
+# Convolutional neural networks 🖥️ for classifying chromatin morphology 🧬 in live cell imaging 🔬
+<br/>
 
-### Convolutional neural networks for classifying chromatin morphology in live cell imaging
+<img width="50%" align="right" alt="Protocol" src="assets/napari_annotator.png" />
 
+Welcome to our [CNN Annotator repository](https://github.com/lowe-lab-ucl/cnn-annotator "CNN Annotator repository || Lowe Lab UCL") where you can find our recent *Methods to Molecular Biology (MiMB) protocol* on how to use a convolutional neural network (CNN) to classify time-lapse microscopy single-cell images patches according to live-cell chromatin morphology.
```
Comment (laureho) [inline README.md]: I think it's _Methods in Molecular Biology_

**Review by laureho — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (laureho) [reply README.md]: Minor thing: @KristinaUlicna if you have time, can you add a white rectangle background to the 'protocol_pipeline.png'? When github is shown in dark mode, one can't see the black arrows. I figure some users may experience that.

**Pull request merged**